<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Board</class>
 <widget class="QWidget" name="Board">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1204</width>
    <height>770</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QTableWidget" name="checkBoard">
     <property name="minimumSize">
      <size>
       <width>1000</width>
       <height>0</height>
      </size>
     </property>
     <property name="rowCount">
      <number>20</number>
     </property>
     <property name="columnCount">
      <number>20</number>
     </property>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <column/>
     <column/>
     <column/>
     <column/>
     <column/>
     <column/>
     <column/>
     <column/>
     <column/>
     <column/>
     <column/>
     <column/>
     <column/>
     <column/>
     <column/>
     <column/>
     <column/>
     <column/>
     <column/>
     <column/>
     <item row="0" column="0">
      <property name="text">
       <string/>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
     </item>
     <item row="0" column="1">
      <property name="text">
       <string/>
      </property>
     </item>
     <item row="0" column="2">
      <property name="text">
       <string/>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
     </item>
     <item row="0" column="3">
      <property name="text">
       <string/>
      </property>
     </item>
     <item row="0" column="4">
      <property name="text">
       <string/>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
     </item>
     <item row="0" column="7">
      <property name="text">
       <string/>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
     </item>
     <item row="0" column="9">
      <property name="text">
       <string/>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
     </item>
     <item row="0" column="10">
      <property name="text">
       <string/>
      </property>
     </item>
     <item row="0" column="11">
      <property name="text">
       <string/>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
     </item>
     <item row="0" column="14">
      <property name="text">
       <string/>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
     </item>
     <item row="0" column="16">
      <property name="text">
       <string/>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
     </item>
     <item row="0" column="18">
      <property name="text">
       <string/>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
     </item>
     <item row="0" column="19">
      <property name="text">
       <string/>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
     </item>
     <item row="1" column="1">
      <property name="text">
       <string> </string>
      </property>
     </item>
     <item row="1" column="2">
      <property name="text">
       <string>西凉城</string>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
     </item>
     <item row="1" column="3">
      <property name="text">
       <string> </string>
      </property>
     </item>
     <item row="1" column="4">
      <property name="text">
       <string>赌场</string>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
     </item>
     <item row="1" column="5">
      <property name="text">
       <string> </string>
      </property>
     </item>
     <item row="1" column="6">
      <property name="text">
       <string> </string>
      </property>
     </item>
     <item row="1" column="7">
      <property name="text">
       <string>长安城</string>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
     </item>
     <item row="1" column="8">
      <property name="text">
       <string> </string>
      </property>
     </item>
     <item row="1" column="9">
      <property name="text">
       <string>井州城</string>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
     </item>
     <item row="1" column="10">
      <property name="text">
       <string> </string>
      </property>
     </item>
     <item row="1" column="11">
      <property name="text">
       <string>洛阳城</string>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
     </item>
     <item row="1" column="12">
      <property name="text">
       <string/>
      </property>
     </item>
     <item row="1" column="13">
      <property name="text">
       <string>商店</string>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
     </item>
     <item row="1" column="14">
      <property name="text">
       <string/>
      </property>
     </item>
     <item row="1" column="15">
      <property name="text">
       <string>冀州城</string>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
     </item>
     <item row="1" column="16">
      <property name="text">
       <string/>
      </property>
     </item>
     <item row="1" column="17">
      <property name="text">
       <string/>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
     </item>
     <item row="1" column="18">
      <property name="text">
       <string/>
      </property>
     </item>
     <item row="1" column="19">
      <property name="text">
       <string/>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
     </item>
     <item row="2" column="1">
      <property name="text">
       <string>起点</string>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
     </item>
     <item row="2" column="3">
      <property name="text">
       <string/>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
     </item>
     <item row="2" column="18">
      <property name="text">
       <string>许昌城</string>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
     </item>
     <item row="3" column="1">
      <property name="text">
       <string/>
      </property>
     </item>
     <item row="3" column="18">
      <property name="text">
       <string/>
      </property>
     </item>
     <item row="4" column="1">
      <property name="text">
       <string/>
      </property>
     </item>
     <item row="4" column="18">
      <property name="text">
       <string>徐州城</string>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
     </item>
     <item row="4" column="19">
      <property name="text">
       <string/>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
     </item>
     <item row="5" column="1">
      <property name="text">
       <string/>
      </property>
     </item>
     <item row="5" column="18">
      <property name="text">
       <string/>
      </property>
     </item>
     <item row="6" column="1">
      <property name="text">
       <string>汉中城</string>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
     </item>
     <item row="6" column="18">
      <property name="text">
       <string/>
      </property>
     </item>
     <item row="7" column="0">
      <property name="text">
       <string/>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
     </item>
     <item row="7" column="1">
      <property name="text">
       <string/>
      </property>
     </item>
     <item row="7" column="18">
      <property name="text">
       <string/>
      </property>
     </item>
     <item row="8" column="1">
      <property name="text">
       <string/>
      </property>
     </item>
     <item row="8" column="18">
      <property name="text">
       <string>酒馆</string>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
     </item>
     <item row="8" column="19">
      <property name="text">
       <string/>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
     </item>
     <item row="9" column="1">
      <property name="text">
       <string/>
      </property>
     </item>
     <item row="9" column="18">
      <property name="text">
       <string/>
      </property>
     </item>
     <item row="10" column="1">
      <property name="text">
       <string/>
      </property>
     </item>
     <item row="10" column="18">
      <property name="text">
       <string/>
      </property>
     </item>
     <item row="11" column="0">
      <property name="text">
       <string/>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
     </item>
     <item row="11" column="1">
      <property name="text">
       <string>商店</string>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
     </item>
     <item row="11" column="18">
      <property name="text">
       <string>商店</string>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
     </item>
     <item row="12" column="1">
      <property name="text">
       <string/>
      </property>
     </item>
     <item row="12" column="18">
      <property name="text">
       <string>建业城</string>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
     </item>
     <item row="12" column="19">
      <property name="text">
       <string/>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
     </item>
     <item row="13" column="1">
      <property name="text">
       <string>酒馆</string>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
     </item>
     <item row="13" column="18">
      <property name="text">
       <string/>
      </property>
     </item>
     <item row="14" column="1">
      <property name="text">
       <string/>
      </property>
     </item>
     <item row="14" column="18">
      <property name="text">
       <string/>
      </property>
     </item>
     <item row="15" column="1">
      <property name="text">
       <string/>
      </property>
     </item>
     <item row="15" column="18">
      <property name="text">
       <string/>
      </property>
     </item>
     <item row="16" column="1">
      <property name="text">
       <string>成都城</string>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
     </item>
     <item row="16" column="18">
      <property name="text">
       <string>柴桑城</string>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
     </item>
     <item row="16" column="19">
      <property name="text">
       <string/>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
     </item>
     <item row="17" column="0">
      <property name="text">
       <string/>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
     </item>
     <item row="17" column="1">
      <property name="text">
       <string/>
      </property>
     </item>
     <item row="17" column="18">
      <property name="text">
       <string/>
      </property>
     </item>
     <item row="18" column="1">
      <property name="text">
       <string/>
      </property>
     </item>
     <item row="18" column="2">
      <property name="text">
       <string>武陵城</string>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
     </item>
     <item row="18" column="3">
      <property name="text">
       <string/>
      </property>
     </item>
     <item row="18" column="4">
      <property name="text">
       <string>商店</string>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
     </item>
     <item row="18" column="5">
      <property name="text">
       <string/>
      </property>
     </item>
     <item row="18" column="6">
      <property name="text">
       <string/>
      </property>
     </item>
     <item row="18" column="7">
      <property name="text">
       <string>襄阳城</string>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
     </item>
     <item row="18" column="8">
      <property name="text">
       <string/>
      </property>
     </item>
     <item row="18" column="9">
      <property name="text">
       <string/>
      </property>
     </item>
     <item row="18" column="10">
      <property name="text">
       <string>赌场</string>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
     </item>
     <item row="18" column="11">
      <property name="text">
       <string>酒馆</string>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
     </item>
     <item row="18" column="12">
      <property name="text">
       <string/>
      </property>
     </item>
     <item row="18" column="13">
      <property name="text">
       <string>荆州城</string>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
     </item>
     <item row="18" column="14">
      <property name="text">
       <string/>
      </property>
     </item>
     <item row="18" column="15">
      <property name="text">
       <string/>
      </property>
     </item>
     <item row="18" column="16">
      <property name="text">
       <string>长沙城</string>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
     </item>
     <item row="18" column="17">
      <property name="text">
       <string/>
      </property>
     </item>
     <item row="18" column="18">
      <property name="text">
       <string/>
      </property>
     </item>
     <item row="19" column="2">
      <property name="text">
       <string/>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
     </item>
     <item row="19" column="4">
      <property name="text">
       <string/>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
     </item>
     <item row="19" column="8">
      <property name="text">
       <string/>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
     </item>
     <item row="19" column="10">
      <property name="text">
       <string/>
      </property>
     </item>
     <item row="19" column="11">
      <property name="text">
       <string/>
      </property>
     </item>
     <item row="19" column="14">
      <property name="text">
       <string/>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
     </item>
     <item row="19" column="17">
      <property name="text">
       <string/>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
     </item>
    </widget>
   </item>
   <item row="0" column="1">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLineEdit" name="playerTurn">
       <property name="minimumSize">
        <size>
         <width>170</width>
         <height>50</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>金钱:</string>
       </property>
       <widget class="QLineEdit" name="currentPlayerMoney">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>133</width>
          <height>20</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>30</height>
         </size>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>310</y>
          <width>178</width>
          <height>298</height>
         </rect>
        </property>
        <property name="title">
         <string/>
        </property>
        <widget class="QWidget" name="layoutWidget">
         <property name="geometry">
          <rect>
           <x>40</x>
           <y>20</y>
           <width>91</width>
           <height>98</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLineEdit" name="diceNumber">
            <property name="minimumSize">
             <size>
              <width>70</width>
              <height>50</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>50</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Arial</family>
              <pointsize>20</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="throwDice">
            <property name="minimumSize">
             <size>
              <width>50</width>
              <height>40</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Arial</family>
              <pointsize>18</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>掷骰子</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="checkInfoBtn">
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>40</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>18</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>查看信息</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="useItemBtn">
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>40</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>18</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>使用道具</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="quitGameBtn">
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>40</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>18</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>退出游戏</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
